src/main/java/com/icthh/xm/uaa/security/DomainTokenServices.java

```diff
@@ -48,6 +49,8 @@
     @Setter
     private TokenStore tokenStore;
     @Setter
+    private XmJwkTokenStore jwkTokenStore;
```

Comment:
@olye0216 @vkirichenko why do we need XmJwkTokenStore here?

Reply:
Currently, there are no needs for XmJwkTokenStore in DomainTokenService, but it placed there for further implementations for stateful mode. It's some kind of "bread crumbs" and can be removed if necessary.

Reply:
XmJwkTokenStore deleted in this class as unused.